src/module-vsf-kco/Model/Klarna/DataTransform/Request/Address.php

```diff
@@ -49,7 +49,7 @@ public function prepareMagentoAddress(DataObject $klarnaAddressDto)
             : $klarnaAddressDto->getCareOf();
 
 
-        $region = $this->region->loadByCode($klarnaAddressDto->getRegion(), $country);
+        $region = $this->region->loadByName($klarnaAddressDto->getRegion(), $country);
```

Comment:
@anhuy1989 Is there a point to keep both of them and use code version as a fallback if `loadByName` doesn't find anything to map against?

Reply:
I fixed @indiebytes .